CMakeLists.txt

```diff
@@ -229,6 +229,15 @@ else()
   set(FLEX_FZNLexer_OUTPUTS ${PROJECT_SOURCE_DIR}/chuffed/flatzinc/generated_parser/lexer.yy.cpp)
 endif()
 
+# If CMAKE_UNITY_BUILD is enabled, we still want these files to be built separately
+# to avoid compilation errors.
+set_source_files_properties(
+    ${BISON_FZNParser_OUTPUTS}
+    ${FLEX_FZNLexer_OUTPUTS}
+  PROPERTIES
+    SKIP_UNITY_BUILD_INCLUSION ON
```

Comment:
I haven't really looked at unity builds yet, but the improvement in build times are promising. Could you explain why these particular files would have to be skipped in the batched builds. Is this because it would otherwise not be guaranteed that the files have been generated?

Reply:
Conflicting definitions within them, so if they both get added to the same compile unit, it isn't happy.